src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdownModalContent.ts

```diff
@@ -0,0 +1,55 @@
+import { IResponsiveDropdownContent, ResponsiveDropdownContent } from './ResponsiveDropdownContent';
+import { Dom, $$, l } from '../../../Core';
+import { SVGIcons } from '../../../utils/SVGIcons';
+import { FocusTrap } from '../../FocusTrap/FocusTrap';
+
+export class ResponsiveDropdownModalContent implements IResponsiveDropdownContent {
+  private closeButton: Dom;
+  private className: string;
+  private focusTrap: FocusTrap;
+
+  constructor(private componentName: string, public element: Dom, private closeButtonLabel: string, private close: () => void) {
+    this.className = `coveo-${this.componentName}-dropdown-modal-content`;
+  }
+
+  private set hidden(shouldHide: boolean) {
+    this.element.toggleClass('coveo-hidden', shouldHide);
+  }
+
+  public positionDropdown(): void {
```

Comment:
You don't necessarily need to type the return value here (void) as well as for the other methods. Typescript will automatically resolve it.

Reply:
Woops, I'm not sure why I specified it. I'll remove unnecessary explicit return types.